python/lsst/ts/electrometer/csc.py

```diff
@@ -30,6 +53,11 @@ class ElectrometerCsc(salobj.ConfigurableCsc):
     event_loop_task : `asyncio.Task`
         A task for handling the event loop.
         Currently not implemented.
+    default_force_output : `bool`
+        Force the output of an event.
+    bucket : `None` or `salobj.AsyncS3Bucket`
+    index : `int`
+        The index of the CSC.
```

Comment:
Please get rid the `index` attribute. You are duplicating information that is available as `self.salinfo.index`